### DISCUSSION TITLE
An image file crashes lime in Firefox.

### DISCUSSION THREAD

**Discussion by TheHENOOB** [HaxeFlixel/flixel#3088] [Q&A] [answered]
The image file in question is this one:

![Track](https://github.com/HaxeFlixel/flixel/assets/74883505/4786085d-db75-4a4e-9a4a-36f52a011777)

I made this image for my game CarTrip and when I put this image on assets and run this game through firefox, it crashes on the preloader.

![image](https://github.com/HaxeFlixel/flixel/assets/74883505/183123bc-2ef9-43d2-8550-4670bee3ffd4)

Any reason why it happens?

**Comment by Vortex2Oblivion** [comment]
Does it do this on other targets?
Also send code

**Reply by TheHENOOB** [reply]
For me, it happens only on HTML5 running on Firefox, Chromium browsers works fine.

I even tested my old build of cartrip on other pcs that have firefox installed, it went the same result.

There's just no code, I picked up the HaxeFlixel template and put the image file on assets/images and the issue persisted.

The origins of this image is from a build that I compiled from Aseprite.
https://github.com/aseprite/aseprite

**Reply by TheHENOOB** [reply]
I suggest you try it out yourself, put the image on the assets folder and test the template on Firefox.

**Comment by Geokureli** [comment, marked as answer]
sometimes browsers block images based on the name, I had this issue with twitter.png and i just had to rename the file
